src/distrs/pois.jl

```diff
@@ -19,6 +19,14 @@ poislogpdf(λ::T, x::T) where {T <: Real} = xlogy(x, λ) - λ - loggamma(x + 1)
 
 poislogpdf(λ::Number, x::Number) = poislogpdf(promote(float(λ), x)...)
 
+poislogpdf(λ::Number, x::Number) = poislogpdf(promote(float(λ), x)...)
+
+function poislogcdf(λ::Number, x::Number)
+    return SpecialFunctions.loggamma(Int(x) + 1, float(λ)) - SpecialFunctions.logfactorial(Int(x))
+end
```

Comment:
It seems a bit restrictive to assume that `x` is an integer:
```suggestion
function poislogcdf(λ::Number, x::Number)
    return loggamma(floor(x + 1), float(λ)) - logfactorial(floor(x))
end
```
Are you sure it is necessary to call `float`?

Reply:
> Are you sure it is necessary to call float?

Nope! I just did because of the code above.

Reply:
Oh and regarding restricting `Int`: I assumed the methods would be more numerically stable when doing so + couldn't see why you'd call these with non-integer types, i.e. I would think if you write `1.0` you actually mean `1`. 

I'm fine with whatever here though!

Reply:
You could want to evaluate the CDF also for e.g. `x=1.5`. This makes sense mathematically and is supported by Rmath.

Reply:
When you say "mathematically", you mean in the sense that "the mathematical function supports it", not in the sense that P(x \le 1.5) \ne P(x \le 1) for a discrete rv. x, right?

Reply:
Ah nvm, you added a `floor` function: I am happy with this:)